src/platform/mhv/secondary-nav/tests/MhvSecondaryNavItem.unit.spec.jsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { expect } from 'chai';
+import MhvSecondaryNavItem from '../components/MhvSecondaryNavItem';
+
+describe('MHV Secondary Navigation Item Component', () => {
+  describe('handle abbreviations', () => {
+    it('when provided', () => {
+      const title = 'a title';
+      const abbr = 'an abbr';
+      const { getAllByText } = render(
+        <MhvSecondaryNavItem
+          title={title}
+          abbreviation={abbr}
+          iconClass="fas fa-home"
+          href="/my-health"
+        />,
+      );
+      expect(getAllByText(title)?.length).to.eql(1);
+      expect(getAllByText(abbr)?.length).to.eql(1);
```

Comment:
Using the optional chaining `?.` operator in a test is a code smell. Tests should be deterministic.

Reply:
Perhaps what you want is something along the lines of:

```javascript
// test case 1, when abbreviation is provided
const abbreviation = "Appts";
const title = "Appointments";
const { querySelector } = render(<MhvSecondaryNavItem abbreviation={abbreviation} title={title}  ... />);
const result = querySelector("abbr").getAttribute('title');
expect(result).to.eq(title);

// test case 2, when abbreviation is not provided
// ...
```